openapi/auth/routes.py

```diff
@@ -15,6 +15,9 @@ def register():
             name = data.get("name")
             email = data.get("email")
             password = data.get("password")
+            confirm_password = data.get("confirm_password")
+            if password != confirm_password:
+                return jsonify({"Error": "Password and confirm_password do not match"})
```

Comment:
This would return an error but still give 200 status ode,Include status code to response

Reply:
Guardian of the codebase

Reply:
I have added the status code